src/frontends/tensorflow_common/src/op/reduce.cpp

```diff
@@ -31,6 +32,25 @@ OutputVector translate_direct_reduce_op(const NodeContext& node) {
     return {reduce_op};
 }
 
+OutputVector translate_prod_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"Prod"}, true);
+    auto input = node.get_input(0);
+    auto axis = node.get_input(1);
+    auto keep_dims = node.get_attribute<bool>("keep_dims", false);
+    auto complex_type_input = as_type_ptr<ComplexTypeMark>(input.get_node_shared_ptr());
+    if (complex_type_input) {
+        input = complex_type_input->input_value(0);
+        auto prod_op = make_shared<v1::ReduceProd>(input, axis, keep_dims);
```

Comment:
please apply update for `axis` if it is negative using `Select` operation like in other PRs you know.